examples/failing-examples/bound-vs-free.golden

```diff
@@ -0,0 +1,4 @@
+#[bound-vs-free.kl:23.25-23.31]<result> : Syntax
+#[bound-vs-free.kl:24.25-24.31]<result> : Syntax
+#[bound-vs-free.kl:25.25-25.31]<result> : Syntax
+#[bound-vs-free.kl:26.25-26.31]<result> : Syntax
```

Comment:
I am glad to see that `do.kl` is now found, but this is still not the output I expected. This is in the `failing-examples` folder, so this example is supposed to fail!

Oh, I see that the example which is supposed to trigger the error is commented-out. Could you please uncomment it?